source/contribute/maintainer.rst

```diff
@@ -57,3 +57,26 @@ the ``use-next-minor-as-default-branch`` switch for the
     $ git switch --create 2.0.x 2.0.0
     Switched to a new branch "2.0.x"
     $ git push doctrine 2.0.x
+
+Tagging a release
+-----------------
+
+We use ``laminas/automatic-releases`` to make releasing as easy as
+closing a milestone. Before closing a milestone, one should check that
+it is complete. If not, they should unassign issues and pull requests,
+or close them / merge them as appropriate. It is also a good idea to
+check which pull requests were merged since the last release on the
+relevant branch, and to make sure each one is properly labelled and
+assigned to the milestone.
+
+Labels are used to generate the release notes, and we should strive to
+provide a consistent experience accross repositories. Here are labels
+that we recommend adding on every repository:
+
+Documentation
+  Includes changes to rst files, markdown files as well as code comments
+  that are not phpdoc comments.
+
+Internal
+  Changes that do not impact the end user, but might impact contributors
+  or maintainers, such as improvements to the CI.
```

Comment:
What other labels should we use across repositories? Should we take this as an occasion to challenge tagging bugfixes with `Bug`?

Right now we have:
- `Bug`, which could also be `Bugfix`, or `Patch`
- `New feature` / `Enhancement`, which could also be `Minor`
- `BC Break`, which could also be `Major`

What naming do you prefer for the three things?

Also, we should probably pick colors for each tag we agree on.

Reply:
`BC Break` can also mark a PR of having an unwanted BC break and once it gets fixed, the label can be removed.

Reply:
But you wouldn't have something tagged as `Major` if there is no BC break, would you?

Reply:
Correct, not the other way around. I see `BC Break` and `Major` with two different meanings.

Reply:
On a historical view you can't tell the difference between a patch or a minor by the branch name. In case you want to tell if a one year old PR was either patch or minor, but I don't know if you ever had such a use case.

Reply:
Why would I need to? In any event, the major/minor/patch terminology primarily belongs to semver. Using the same terminology for changes, especially if it doesn't exactly match the kind of the version where this change was made will make it hard to comprehend.

Reply:
Okay, if this info isn't needed later on, we don't need the release-labels.